src/aktualizr_secondary/secondary_tcp_server.cc

```diff
@@ -159,6 +167,7 @@ bool SecondaryTcpServer::HandleOneConnection(int socket) {
         if (install_result == data::ResultCode::Numeric::kNeedCompletion) {
           need_reboot = true;
         }
+        LOG_INFO << "Installation complete" << (need_reboot ? ", but needs reboot to complete." : "!");
```

Comment:
What's wrong with logs in  'AktualizrSecondary::install', this log just duplicates what is in the install() method?

Reply:
Ah, yes, this is duplicated with existed log. I will remove it.